src/AccordionPanel.jsx

```diff
@@ -0,0 +1,198 @@
+			open: newState
+		});
+
+	}
+
+	/**
+	 * @returns {undefined}
+	 *
+	 * Sets height of `AccordionPanel` to be appear open or closed when mounting
+	 */
+	componentDidMount() {
+		this.setState({
+			height: `${this.state.open * this.content.getBoundingClientRect().height}px`
+		});
+	}
+
+	/**
+	 * @returns {undefined}
+	 *
+	 * Updates state to toggle `AccordionPanel` open and closed
+	 */
+	componentWillUpdate(nextProps, nextState) {
+		if (nextProps.isOpen !== this.state.open) {
+			this.setState({
+				open: nextProps.isOpen,
+				height: `${nextProps.isOpen * this.content.getBoundingClientRect().height}px`
+			});
+		}
+	}
+
+	/**
+	 * @returns {String} icon shape
+	 */
+	getIconShape() {
+		const {
+			iconShape,
+			iconShapeActive
+		} = this.props;
+
+		return this.state.open && iconShapeActive ?
+			iconShapeActive :
+			iconShape;
+	}
+
+	render() {
+		const {
+			panelContent,
+			label,
+			isOpen, // eslint-disable-line no-unused-vars
+			setClickedPanel, // eslint-disable-line no-unused-vars
+			iconAlign, // eslint-disable-line no-unused-vars
+			iconShape, // eslint-disable-line no-unused-vars
+			iconSize,  // eslint-disable-line no-unused-vars
+			iconShapeActive, // eslint-disable-line no-unused-vars
+			isAnimated,
+			classNamesActive,
+			className,
+			...other
+		} = this.props;
+
+		const classNames = {
+			accordionPanel: cx(
+				PANEL_CLASS,
+				{
+					[ACTIVEPANEL_CLASS]: this.state.open,
+					[classNamesActive]: this.state.open && classNamesActive
+				},
+				className
+			),
+			trigger: cx(
+				className,
+				'accordionPanel-label display--block span--100'
+			),
+			content: cx(
+				{
+					'accordionPanel-animator': isAnimated,
+					'accordionPanel-animator--collapse': !this.state.open,
+					'visibility--a11yHide': !isAnimated && !this.state.open
+				}
+			)
+		};
+
+		// create valid attribute name from trigger label
+		const ariaId = label.replace(/\s+/g, '').toLowerCase();
+
+		return(
+			<Flex
+				className={classNames.accordionPanel}
+				rowReverse={iconAlign === 'left' && 'all'}
+				{...other}
+				>
+
+				<FlexItem>
+					<Chunk>
+						<button
+							role='tab'
+							id={`label-${ariaId}`}
+							aria-controls={`panel-${ariaId}`}
+							aria-expanded={this.state.open}
+							aria-selected={this.state.open}
+							className={classNames.trigger}
+							onClick={this._handleToggle}>
+								{label}
+						</button>
+					</Chunk>
+
+					<Chunk
+						role='tabpanel'
+						aria-labelledby={`label-${ariaId}`}
+						aria-hidden={!this.state.open}
+						className={classNames.content}
+						style={{height: this.state.height}}>
+						<div
+							className='accordionPanel-content'
+							ref={(div) => { this.content = div; }}>
+							{panelContent}
+						</div>
+					</Chunk>
+				</FlexItem>
+
+				<FlexItem
+					className='accordionPanel-icon'
+					onClick={this._handleToggle}
+					shrink
+					>
+					<Icon
+						shape={this.getIconShape()}
+						size={iconSize} />
```

Comment:
align closing `>` with opening `<Icon`, or just make the whole opening tag inline - it's short

```jsx
<Icon shape={this.getIconShape()} size={iconSize} />
```